test/applications/cmd/monitoring-dns-nsc/corefile/corefile.go

```diff
@@ -0,0 +1,196 @@
+package corefile
```

Comment:
@denis-tingajkin did you try to search existing library for this?

Reply:
@haiodo yes, I tried to search and did not find anything similar.